src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -158,4 +161,8 @@ public Request getHighestPriorityFulfillableRequest() {
   public String getLoggedInUserId() {
     return loggedInUserId;
   }
+
+  public DateTime getSystemDateTime() {
```

Comment:
I understand this to be the system return date time, rather than the general system date time.

```suggestion
  public DateTime getSystemReturnDateTime() {
```

Reply:
Sure.